recognition/app/main.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import requests
+from flask import Flask
+
+from app.domain.securityrule.core import IntrusionRule
+from app.domain.securityrule.utils.utils import is_intrusion_rule_active
+from app.presentation.securityrule.IntrusionRuleSerializer import (
+    IntrusionRuleSerializer,
+)
+from app.recognizer.RecognizersManager import RecognizersManager
+from app.utils.Logger import logger
+from app.utils.env import DEV_API_KEY, ALARM_PORT, ALARM_HOST
+from app.utils.interval import set_interval
+
+intrusion_rules: List[IntrusionRule] = []
+manager = RecognizersManager()
+
+
+def create_app():
+    app = Flask(__name__)
+    intrusion_rules.append(*get_intrusion_rules())
+    logger.info(intrusion_rules)
+    enable_intrusion_rules()
+
+    set_interval(check_rule_update, seconds=60)
+    set_interval(check_rule_validity, seconds=60)
+    return app
+
+
+def get_intrusion_rules() -> List[IntrusionRule]:
+    url = f"http://{ALARM_HOST}:{ALARM_PORT}/security-rules/intrusions"
+    headers = {"Authorization": f"Bearer {DEV_API_KEY}"}
+    res = requests.get(url, headers=headers)
+    rules: List[IntrusionRule] = []
```

Comment:
a manager would be great but i understand. we can keep this :D

Reply:
I know...